README.md

```diff
@@ -142,31 +146,48 @@ To get started, add [Sourcery to your GitHub repo](https://github.com/apps/sour
 
 Not everyone's coding style is the same, so we know you won't always want to accept every refactoring Sourcery suggests.
 
-Generally, for configurations, Sourcery uses configuration settings from `.sourcery.yaml` in the project directory.
+Sourcery reads [configuration settings from two sources](https://docs.sourcery.ai/Reference/Configuration/Sources/#configuration-sources):
+
+- a [`.sourcery.yaml` file](https://docs.sourcery.ai/Reference/Configuration/sourcery-yaml/) in the project directory; and
+- a [user-specific config file](https://docs.sourcery.ai/Reference/Configuration/Sources/#user-configuration-file-system-locations).
+
+You can use the project-specific config file to set project-specific settings, and the user-specific config file to set settings that apply to all projects.
```

Comment:
I think it's also important to tell that project-specific config file takes precedence over user-specific.
I guess it's intuitive, but in my opinion it's better to be explicit in readme file.

Reply:
Great call! Done in 774b471854f8f1038dcafa08ac94db416b3f085e.